ui/app/src/components/shared/TopNav.tsx

```diff
@@ -10,7 +10,10 @@ export const TopNav: React.FunctionComponent = () => {
       <div className={contentClass}>
         <Stack horizontal>
           <Stack.Item grow={100}>
-            <Link to='/' className='tre-home-link'>Azure Trusted Research Environment</Link>
+            <Link to='/' className='tre-home-link'>
+              <Icon iconName="TestBeakerSolid" style={{ marginLeft: '10px', marginRight: '10px', verticalAlign: 'middle' }} />
+              <h5 style={{display: 'inline'}}>Azure TRE</h5>
```

Comment:
@marrobi are you ok with the main page title being `Azure TRE`, or do you want the full title `Azure Trusted Research Environment` ? I'm just aware that we don't have the full title anywhere, will a new user know what `TRE` means?

Reply:
Ideally it would be pulled in from a config value of some sort. Is this possible?

Reply:
in future yea, for now i'm switching it back to the full title